README.md

```diff
@@ -1,102 +1,95 @@
-# lint-prepush    
+# lint-prepush
```

Comment:
There were some extra spaces at the end of this line and the new line below it just gives things a little room for easier scanning.